.github/workflows/publish.yml

```diff
@@ -105,3 +105,9 @@ jobs:
         env:
           NODE_AUTH_TOKEN: ${{ secrets.NPM_PUBLISH }}
         run: npm publish --access public
+
+      - name: Create GitHub Release
+        uses: softprops/action-gh-release@v1
+        if: github.event_name == 'workflow_dispatch'
+        with:
+          files: ./dist/*
```

Comment:
@simlay as I understand this will add the `index.node` files inside `dist/`.
I think we would have to build from a matrix on a different step if we want to have zip file with build artifacts per platform.

WDYT?

Reply:
Yeah. The `index.node` is basically a DLL. The release CI builds one for each architecture we support. The macOS one is bundled together with [`lipo(1)`](https://www.unix.com/man-page/OSX/1/lipo/). Then at runtime, the node client loads the correct file. I'm not quite sure what else is needed but that's definitely required.